logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/SendMessagePartialFailureMapper.kt

```diff
@@ -74,6 +74,7 @@ internal class SendMessagePartialFailureMapperImpl : SendMessagePartialFailureMa
 
 data class MessageSent(
     val time: String,
-    val failed: List<UserId> = listOf(),
+    val failedToConfirmClients: List<UserId> = listOf(),
+    val failedToSend: List<UserId> = listOf(),
```

Comment:
Why expose it if we don't map and don't use it? Expose it when you start using it.

Reply:
Fair point, I removed it :D